File: python/Programmers_level/level1/제일 작은 수 제거하기.py
def solution(arr):
    # list.remove()
    mini = min(arr) # 최소값 찾기
    arr.remove(mini) # 배열에서 최소값 제거
    
    if len(arr) == 0: # 빈 배열 예외처리
        arr.append(-1)
        
    answer = arr
    return answer

File: Javascript/Programmers_level/level1/제일 작은 수 제거하기.js
function solution(arr) {
  const d = Math.min(...arr)
  let answer = arr.filter((num) => num !== d)
  if (answer.length === 0) {
    answer.push(-1)
  }
  return answer
}

/* 
다른 사람의 풀이
function solution(arr) {
    arr.splice(arr.indexOf(Math.min(...arr)),1);
    if(arr.length<1)return[-1];
    return arr;
} 
*/
